jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -256,75 +267,45 @@ public boolean isContainedIn(Resource r)
     @Override
     public URI getTargetURI()
     {
-        if (!targetResolved)
-        {
-            targetPath = resolveTargetPath();
-            targetResolved = true;
-        }
-        if (targetPath == null)
-            return null;
+        checkAlias();
         return targetPath.toUri();
     }
 
-    private Path resolveTargetPath()
+    private void checkAlias()
```

Comment:
Rename to `resolveAlias()`.

Reply:
I agree with @sbordet, this is a `resolveAlias()` or `resolveTarget()` name.